pymc3/sampling.py

```diff
@@ -81,7 +81,7 @@ def assign_step_methods(model, step=None, methods=(NUTS, HamiltonianMC, Metropol
     return steps
 
 
-def sample(draws, step=None, init='advi', n_init=200000, start=None,
+def sample(draws, step=None, n_advi=200000, n_nuts=2000, start=None,
```

Comment:
Yep exactly. There could be different init-functions and we always forward **init_params.